Cargo.toml

```diff
@@ -54,15 +53,29 @@ digest = { version = "0.10", default-features = false, optional = true }
 subtle = { version = "2.3.0", default-features = false }
 serde = { version = "1.0", default-features = false, optional = true, features = ["derive"] }
 zeroize = { version = "1", default-features = false, optional = true }
+unsafe_target_feature = { version = "= 0.1.1", optional = true }
+
+[target.'cfg(target_arch = "x86_64")'.dependencies]
+cpufeatures = "0.2.6"
 
 [target.'cfg(curve25519_dalek_backend = "fiat")'.dependencies]
 fiat-crypto = "0.1.19"
 
 [features]
-default = ["alloc", "precomputed-tables", "zeroize"]
+default = ["alloc", "precomputed-tables", "zeroize", "simd"]
```

Comment:
Perhaps AVX2 should also be enabled by default, since it works on `stable`?

Reply:
> Perhaps AVX2 should also be enabled by default, since it works on `stable`?

The `simd` features enables both of them. (: 

```
simd = ["simd_avx2", "simd_avx512"]
```

Reply:
Doesn't AVX-512 need nightly?

Reply:
It does, but nightly is detected separately. It needs *both* the `simd_avx512` feature *and* to be on nightly to activate.